### PR TITLE
Chat input mode should change to enabled if operator sends message first

### PR DESCRIPTION
**Pull request by deniss-mc**
**Jira issue:**
https://glia.atlassian.net/browse/MUIC-643

### CODE REVIEW

File: widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java [hunk 1]
```diff
@@ -747,19 +747,29 @@ private void appendMessageItem(List<ChatItem> currentChatItems, ChatMessage mess
...
+
+    private void onOperatorMessageReceived(List<ChatItem> currentChatItems, ChatMessage message) {
+        changeLastOperatorMessages(currentChatItems, message);
+        appendMessagesNotSeen();
+        changeChatInputMode(message);
+    }
+
+    private void changeChatInputMode(ChatMessage message) {
+        ChatInputMode newInputMode = getChatInputMode(message);
+        if (chatState.chatInputMode != newInputMode)
```
Comment (jakob-mc) [inline widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java:762]: should use curly braces in multiline if-else blocks. official standard.

**Review by jakob-mc — APPROVED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java (hunk 1 above)
Comment (jakob-mc) [reply widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java:762]: https://kotlinlang.org/docs/coding-conventions.html#control-flow-statements

**Review by dukhovnyi — APPROVED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java (hunk 1 above)
Comment (deniss-mc) [reply widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java]: fixed (also for chatinputmode method)

Comment (jakob-mc) [reply widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java]: beautiful

**Pull request merged**